data-catalog-backend-app/src/main/java/no/nav/data/catalog/backend/app/informationtype/domain/InformationProducerCode.java

```diff
@@ -0,0 +1,11 @@
+package no.nav.data.catalog.backend.app.informationtype.domain;
+
+
+public enum InformationProducerCode {
```

Comment:
Skal dette være enums, eller skal kodeverkstabeller? Bør de også vedlikeholdes fra fagsystem/et eller annet register

Reply:
Slik jeg har skjønt det skal alt ligge i den felles kodeverkstabellen, codelist. 
Har en rest-controller for å håndtere dette i front-end men vet ikke om vi skal hente inn verdier fra noe register. 
Kan følge opp det etter påsken.